evm/src/cpu/kernel/tests/mpt/delete.rs

```diff
@@ -48,6 +49,32 @@ fn mpt_delete_branch_into_hash() -> Result<()> {
     test_state_trie(state_trie, nibbles_64(0xADE), test_account_2())
 }
 
+#[test]
+fn test_after_mpt_delete_extension_branch() -> Result<()> {
+    let hash = Node::Hash(H256::random());
+    let branch = Node::Branch {
+        children: std::array::from_fn(|i| {
+            if i == 0 {
+                Node::Empty.into()
+            } else {
+                hash.clone().into()
+            }
+        }),
+        value: vec![],
+    };
+    let nibbles = Nibbles::from_bytes_be(&random::<[u8; 5]>()).unwrap();
+    let state_trie = Node::Extension {
+        nibbles,
+        child: branch.into(),
+    }
+    .into();
+    let key = nibbles.merge_nibbles(&Nibbles {
+        packed: U512::zero(),
+        count: 64 - 2 * 5,
```

Comment:
I'm struggling to understand the `- 2 * 5` here. Is there a specific reason for it?

Reply:
`2*5` is just `nibbles.count`, so we basically add `64-nibbles.count` zeros to `nibbles` to get the key.
I replaced `2*5` with `nibbles.count` to make this clearer.